cart.rb

```diff
@@ -1,13 +1,31 @@
+# frozen_string_literal: true
+
 require 'pry'
 
 class Cart
+  attr_accessor :products_in_cart
+  attr_reader :cart_errors
+
   def initialize
```

Comment:
nie wiem czy biznesowo to jest sensowne, ale na pewno w testach by pomogła możliwość inicjowania Carta z produktami, czyli `def initialize(products_in_cart=nil)`